src/qibolab/transpilers/__init__.py

```diff
@@ -1 +1,5 @@
-from qibolab.transpilers.transpile import can_execute, transpile
+from qibolab.transpilers.fusion import FusionTranspiler, RearrangeTranspiler
+from qibolab.transpilers.gate_decompositions import NativeGateTranspiler
+from qibolab.transpilers.general_connectivity import GeneralConnectivityTranspiler
+from qibolab.transpilers.sequential import SequentialTranspiler
+from qibolab.transpilers.star_connectivity import StarConnectivityTranspiler
```

Comment:
```suggestion
from qibolab.transpilers.fusion import Fusion, Rearrange
from qibolab.transpilers.gate_decompositions import NativeGate
from qibolab.transpilers.general_connectivity import GeneralConnectivity
from qibolab.transpilers.sequential import Sequential
from qibolab.transpilers.star_connectivity import StarConnectivity
```

Since they are all inside the `transpiler` subpackage, it makes sense they are all `Transpiler`s.
When you use them outside, either is very clear that they are what they are, or you can keep the `transpilers.` namespace (`transpilers.StarConnectivity` definitely looks like a transpiler).

Reply:
Good points, thanks. I updated all the names, looks better now.